Backend/src/main/java/com/project/omega/service/implmentations/OrderServiceImpl.java

```diff
@@ -32,7 +37,7 @@ public Order createOrder(Order order) {
     public Order getOrderById(Long id) throws NoRecordsFoundException {
             Optional<Order> order = orderRepository.findById(id);
             if (!order.isPresent()){
-                throw new NoRecordsFoundException("Order with id" + id + " not found");
+                throw new NoRecordsFoundException(messages.getMessage("orderNotFound" , null, null));

```

Comment:
I believe that the message string actually requires a .message prefix like in the .properties file. So "message.orderNotFound".